sections/04_study.md

```diff
@@ -241,114 +241,61 @@ this was easy to simulate with a computational model. As we learn to better
 visualize and analyze the hidden nodes within deep learning models, our
 understanding of TF binding motifs and dynamics will likely improve.
 
-### Promoters, enhancers, and related epigenomic features
-
-Transcriptional control is undoubtedly a vital, early part of the regulation of
-gene expression. An abundance of sequence and associated functional data (e.g.
-ENCODE [@tag:Consortium2012_encode] and ExAC [@doi:10.1038/nature19057]) exists
-across species. At the same time, studies of gene regulation have often focused
-on the protein (binding) rather than the promoter level
-[@doi:10.1093/bib/4.1.22], perhaps due to the ill-defined nature of
-_cis_-regulatory elements (CREs). A promoter itself can be seen as an assemblage
-of "active" binding sites for transcription factors interspersed by
-less-characterized and perhaps functionally silent spacer regions. However, the
-sequence signals that control the start and stop of transcription and
-translation are still not well understood, compounded by incomplete
-understanding of alternative transcripts and the context for these alternatives.
-Sequence similarity is poor even between functionally correlated genes. While
-homologs might be studied for insight, they may not exist or may be just as
-poorly characterized.
-
-Recognizing enhancers presents additional challenges. Enhancers may be up to one
-million base pairs upstream or downstream from the affected promoter on either
-strand and even within the introns of other genes [@doi:10.1038/nrg3458]. They
-do not necessarily operate on the nearest gene and may affect multiple genes.
-Their activity is frequently tissue- or context-specific. Many
-enhancers display modest or no conservation across species. There is no
-universal enhancer sequence signal or marker for enhancers, and some literature
-suggests that enhancers and promoters may be just categories along a spectrum
-[@doi:10.1016/j.tig.2015.05.007]. One study [@doi:10.1101/gr.173518.114] even
```

Comment:
Yes, I cut it intentionally. That study evaluates unsupervised methods for pattern identification and characterization as if they were supervised classifiers, inappropriately.